build/Publish.targets

```
@@ -174,6 +174,10 @@
       <PackageToPublish Include="%(_SymbolsPackageArtifactInfo.ArtifactPath)" Category="symbols" IsSymbolsPackage="true" />
     </ItemGroup>
 
+    <ItemGroup>
+      <RedistPackageToPublish Include="$(InstallersOutputPath)(SharedFrameworkRedistPackageId)*" />
```

Publish job is currently failing because this directory does not exist. Artifacts from earlier builds don't land in the same directory in later ones.
```suggestion
      <RedistPackageToPublish Include="$(DependencyAssetsDir)(SharedFrameworkRedistPackageId)*" />
```

If you prefer to use `$(DependencyPackagesDir)` instead, update the change I made at the bottom of the http://aspnetci/admin/editDependencies.html?id=buildType:Releases_21Public_Publish#artifactDeps table.